bin/cron/build_contact_rollups

```diff
@@ -3,11 +3,43 @@
 require_relative('../../dashboard/config/environment')
 require 'cdo/contact_rollups'
 require 'cdo/contact_rollups_validation'
+require 'cdo/pardot'
+require 'cdo/chat_client'
 require 'cdo/only_one'
 
+CHAT_CHANNEL = "cron daily".freeze
```

Comment:
Good point. I had copied that from other code that Slacked to #cron-daily and tested it and it works. But it is clearer as `"cron-daily"`. Tested that and that works as well. Changed.